composer.json

```
@@ -15,28 +15,28 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^8.0",
         "symfony-cmf/routing": "^2.3.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0"
+        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0"
     },
     "require-dev": {
         "jackalope/jackalope-doctrine-dbal": "^1.3",
         "doctrine/phpcr-odm": "^1.4|^2.0",
-        "symfony/phpunit-bridge": "^5.0",
+        "symfony/phpunit-bridge": "^5.0 || ^6.0",
```

```suggestion
        "symfony/phpunit-bridge": "^6.0",
```

the bridge is only to help with phpunit, no need to support multiple versions.